modules/autokey/main.tf

```diff
@@ -39,7 +39,7 @@ resource "google_kms_key_handle" "primary" {
   provider = google-beta
 
   project                = each.value.project
-  name                   = "${each.value.name}-${random_string.suffix[0].result}"
+  name                   = each.value.name
```

Comment:
We don't actually need to append the random here. I'm removing in order to turn the import process feasible.

Reply:
Hmmm you're right... good catch, thanks!
I'll need to modify the import instructions then. User will need to manually append the random suffix into `.tfvars`, otherwise the resource will be created instead of imported.